### DISCUSSION TITLE
LIKE filtering on the message field is not working

### DISCUSSION THREAD

**Discussion by jorgeaponte** [AmrDeveloper/GQL#125] [Q&A] [answered]
I have a query similar to the following but I have not been able to filter using LIKE on the message field (it works fine on the title field).

```sql
SELECT title, author_name, datetime, message FROM commits WHERE datetime > '2024-09-11 01:01:01' AND message LIKE '%alternative%' 
```

**Comment by AmrDeveloper** [comment, marked as answer]
Hello @jorgeaponte,

Thank you for reporting, the problem was that regex was not supporting multi lines strings like message because that it works fine with title, i updated it to support multi lines and unicode, your query will work fine now, also you can write query like this 

```sql
select count() from commits where message like "%😎%"
```

Solved here: https://github.com/AmrDeveloper/GQL/commit/8a4bebb7745ef05f6410186da135c305184ffbcc

I will release it in `0.29.0`

Thank you, have a nice day
Amr Hesham

**Reply by jorgeaponte** [reply]
Thanks a lot
